backends/tfhe-cuda-backend/cuda/src/crypto/torus.cuh

```diff
@@ -1,6 +1,7 @@
 #ifndef CNCRT_TORUS_CUH
 #define CNCRT_TORUS_CUH
 
+#include "polynomial/parameters.cuh"
```

Comment:
Do we really need this new import?

Reply:
we need it for `get_two_pow_torus_bits` also `parameters.cuh` might not be the best place for this function so we can move it somewhere else.

Reply:
Oh maybe having it in torus.cuh would make more sense?